formats/cbor/commonMain/src/kotlinx/serialization/cbor/Cbor.kt

```diff
@@ -27,24 +24,49 @@ import kotlinx.serialization.modules.*
  * @param encodeDefaults specifies whether default values of Kotlin properties are encoded.
  *                       False by default; meaning that properties with values equal to defaults will be elided.
  * @param ignoreUnknownKeys specifies if unknown CBOR elements should be ignored (skipped) when decoding.
+ * @param writeKeyTags Specifies whether tags set using the [KeyTags] annotation should be written (or omitted)
+ * @param writeValueTags Specifies whether tags set using the [ValueTags] annotation should be written (or omitted)
+ * @param verifyKeyTags Specifies whether tags preceding map keys (i.e. properties) should be matched against the
+ *                      [KeyTags] annotation during the deserialization process. Useful for lenient parsing
+ * @param verifyValueTags Specifies whether tags preceding values should be matched against the [ValueTags]
+ *                      annotation during the deserialization process. Useful for lenient parsing.
+ * @param alwaysUseByteString Specifies whether to always use the compact [ByteString] encoding when serializing
+ *                            or deserializing byte arrays.
+ *
+ * @param writeDefiniteLengths Specifies whether the definite length encoding should be used (as required for COSE, for example)
+ * @param preferCborLabelsOverNames Specifies whether to serialize element labels (i.e. Long from [CborLabel])
+ *                                    instead of the element names (i.e. String from [SerialName]) for map keys
  */
 @ExperimentalSerializationApi
 public sealed class Cbor(
     internal val encodeDefaults: Boolean,
     internal val ignoreUnknownKeys: Boolean,
+    internal val writeKeyTags: Boolean,
+    internal val writeValueTags: Boolean,
+    internal val verifyKeyTags: Boolean,
+    internal val verifyValueTags: Boolean,
+    internal val writeDefiniteLengths: Boolean,
+    internal val preferCborLabelsOverNames: Boolean,
+    internal val alwaysUseByteString: Boolean,
     override val serializersModule: SerializersModule
 ) : BinaryFormat {
 
     /**
      * The default instance of [Cbor]
      */
-    public companion object Default : Cbor(false, false, EmptySerializersModule())
+    public companion object Default :
+        Cbor(false, false, true, true, true, true, false, true, false, EmptySerializersModule())
```

Comment:
I see that you enabled new flags (write/verifyTags, preferCborLabels) by default. We can't do that, because it will change output format in the right moment users will get the new version of the library without their explicit consent